Tests/js/views/assets/ez-usermenuview-tests.js

```diff
@@ -81,18 +81,26 @@ YUI.add('ez-usermenuview-tests', function (Y) {
             var view = this.view,
                 userMenuView = this.userMenuItemSecondView,
                 eventName = 'logout',
-                isEventFired = false;
+                isEventFired = false,
+                isAddedEventFired = false;
 
             view.on('*:' + eventName, function () {
                 isEventFired = true;
             });
 
+            userMenuView.on('addedToUserMenu', function (event) {
+                isAddedEventFired = true;
+
+                Y.Assert.areSame(view, event.userMenu, 'Should send correct parameter');
```

Comment:
The error message is not precise enough. imagine if the test is failing in 6 months, what does *correct parameter* mean? The message should be something like *The user menu view should be available in the event parameters*.